geomesa-hbase/geomesa-hbase-tools/src/main/scala/org/locationtech/geomesa/hbase/tools/HBaseDataStoreCommand.scala

```diff
@@ -18,6 +19,11 @@ trait HBaseDataStoreCommand extends DataStoreCommand[HBaseDataStore] {
 
   override def params: CatalogParam
 
-  override def connection: Map[String, String] =
-    Map(HBaseDataStoreFactory.Params.BigTableNameParam.getName -> params.catalog)
+  override def connection: Map[String, String] = {
+    val remote = GeoMesaSystemProperties.SystemProperty("geomesa.hbase.remote.filtering", "false").get.toBoolean
```

Comment:
instead of doing the lookup here, we should add it in the HBaseDataStoreFactory so that it's usable outside tools